fluffy/component/highlighting.py

```diff
@@ -178,6 +179,23 @@ def get_highlighter(text, language, filename):
     return PygmentsHighlighter(lexer)
 
 
+def create_diff(text, text2):
+    """
+    The diff between two texts is calculated
+    """
```

Comment:
```suggestion
def create_diff(text: str, text2: str) -> typing.Tuple[str, str]:
    """The diff between two texts is calculated."""
```

* Updating docstring for https://peps.python.org/pep-0257/#one-line-docstrings
* Adding type annotations (I know most of fluffy doesn't have them yet)